{"name": "test_xfail", "status": "passed", "description": "\n    >>> import pytest\n\n    >>> @pytest.mark.xfail()\n    ... def test_xfail_example():\n    ...     assert False\n\n    ", "attachments": [{"name": "stdout", "source": "6364acba-b18e-4063-a5ab-507ef9bc474f-attachment.txt", "type": "text/plain"}], "start": 1710872970190, "stop": 1710872970209, "uuid": "95f2b1ec-c0fa-41f6-8332-98023ba67a0d", "historyId": "1abd07900c741abf720a316ad3f57f9e", "testCaseId": "1abd07900c741abf720a316ad3f57f9e", "fullName": "tests.allure_pytest.acceptance.status.xfail_call_status_test#test_xfail", "labels": [{"name": "parentSuite", "value": "tests.allure_pytest.acceptance.status"}, {"name": "suite", "value": "xfail_call_status_test"}, {"name": "host", "value": "MacBook-Pro-whenessel.local"}, {"name": "thread", "value": "29518-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.allure_pytest.acceptance.status.xfail_call_status_test"}]}